src/client/src/App/Credit/SellSide/utils/getStatusColor.ts

```diff
@@ -0,0 +1,24 @@
+import { DefaultTheme } from "styled-components"
+import { SellSideQuoteState } from "../sellSideState"
+
+export const getSellSideStatusColor = (
+  status: SellSideQuoteState,
+  theme: DefaultTheme,
+) => {
+  switch (status) {
+    case SellSideQuoteState.Accepted:
+      return theme.accents.positive.base
+    case SellSideQuoteState.Lost:
+    case SellSideQuoteState.Rejected:
+    case SellSideQuoteState.Passed:
```

Comment:
not convinced Pass is negative, but there we go 😄

Reply:
Yeah, they'll probably see that once it's implemented.  It's like, you did it to yourself!